code/game/objects/items/tanks/tanks.dm

```diff
@@ -213,9 +205,8 @@
 
 /obj/item/tank/assume_air(datum/gas_mixture/giver)
 	air_contents.merge(giver)
-
-	check_status()
-	return 1
+	handle_tolerances(1)
```

Comment:
What's with the fixed delta time?

Reply:
It's because this is event based and I need some sort of scaling factor so extreme pressure/temperatures can actually have an effect.

Reply:
Can you make this a define then instead of a magic number?

Reply:
Sure

Reply:
Done